Sources/XcbeautifyLib/OutputHandler.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+
+
+public class OutputHandler {
+    let quiet: Bool
+    let quieter: Bool
+    let isCI: Bool
+    let writer: (String) -> Void
+    
+    /// In quiet mode, lastFormatted will record last output and whether it will be printed is determined by the current output type. In this way, if we encounter warnings or errors, we get a chance to print last output as their banner. So now the output is in following form:
```

Comment:
Can you wrap lines to 80 characters or less? (not a hard-requirement but the current length is too long)